io-ballerina/line_stream.bal

```diff
@@ -13,25 +13,24 @@
 // KIND, either express or implied.  See the License for the
```

Comment:
```suggestion
// KIND, either express or implied.  See the License for the
```
Please see above.

Reply:
This is the standard license template we have been using.